zanata-war/src/main/java/org/zanata/exception/InvalidApiKeyException.java

```diff
@@ -0,0 +1,43 @@
+package org.zanata.exception;
+
+/**
+ * Exception for invalid API key.
+ *
+ * @author Alex Eng <a href="mailto:aeng@redhat.com">aeng@redhat.com</a>
+ */
+public class InvalidApiKeyException extends Exception {
```

Comment:
Since we never create an instance of this exception, it might be better to call this class `InvalidApiKeyUtil` and put it in the `org.zanata.rest` package.  

I'm sure there's a better way of handing it than yet another util class (which is what this exception really is), but I can't think of it right now.  (It probably involves code generation to enable type-safe formatting for messages.)